static/app/components/charts/worldMapChart.tsx

```diff
@@ -1,11 +1,11 @@
-import * as React from 'react';
-import {withTheme} from '@emotion/react';
+import {useTheme} from '@emotion/react';
 import type {MapSeriesOption, TooltipComponentOption} from 'echarts';
 import * as echarts from 'echarts/core';
 import max from 'lodash/max';
 
+import countryToCodeMap from 'sentry/data/countryCodesMap';
```

Comment:
This will now be directly injected into the final bundle, we should check for bundle size changes and possibly revert this back to async loading if there's a large increase

Reply:
@priscilawebdev Just checked, this is 2.3KB gzipped w gzip default compression. I'm fine with checking this in